recipes/protex/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "protex" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/florianjoerg/protex/archive/v{{ version }}.tar.gz
+  sha256: 39d22c16285ae98c74134f87d5efd13c33bbb0f9cee4c030cf72afe00af14a84
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - openmm =7.6
+    - scipy
+    - numpy
+    - pyyaml
+    - parmed
+    - pytest
+    - pytest-cov
+    - codecov
```

Comment:
Does *every* user of protex really need this testing stuff installed?

Reply:
No, thats not necessary. I removed the dependencies